runtime/doc/api.txt

```diff
@@ -2172,13 +2232,21 @@ nvim_buf_get_mark({buffer}, {name})                      *nvim_buf_get_mark()*
 
                 Marks are (1,0)-indexed. |api-indexing|
 
+                Note:
+                    Returns (0, 0) if the mark is not set, or if the mark is
+                    uppercase/file mark and it doesn't belong to the buffer
```

Comment:
thanks for noting this. let's move it to the `@returns` docstring though. Same for the other cases.